packages/@aws-cdk/core/lib/bundling.ts

```diff
@@ -175,6 +174,27 @@ export class BundlingDockerImage {
       ],
     });
   }
+
+  /**
+   * Copies a file or directory out of the Docker image to the local filesystem
+   */
+  public copyOut(imagePath: string, outputPath: string) {
```

Comment:
Should we rename to `cp` so it's aligned with the `docker` API?

Reply:
Ok, I've pushed up this change.